src/main/resources/META-INF/plugin.xml

```diff
@@ -45,6 +45,12 @@
             <add-to-group group-id="Vcs.Log.ChangesBrowser.Toolbar" anchor="first"/>
         </action>
 
+        <action id="actions.ToggleLabelsAction"
+                class="org.jetbrains.research.refactorinsight.actions.ToggleLabelsAction"
+                text="Show Refactoring Labels" description="Adds the refactoring icon to commits containing refactorings">
+            <add-to-group group-id="Vcs.Log.PresentationSettings" anchor="first"/>
```

Comment:
Could we place the toggle above all "Show <something>" actions in the list?

Reply:
Done